pymeasure/instruments/temptronic/temptronic_base.py

```diff
@@ -125,6 +125,12 @@ class ATSBase(SCPIUnknownMixin, Instrument):
     """The base class for Temptronic ATSXXX instruments.
     """
 
+    def __init__(self, adapter, name="ATSBase", **kwargs):
+        super().__init__(adapter, name=name, **kwargs)
+
+    def wait_for(self, query_delay=None):
+        super().wait_for(0.05 if query_delay is None else query_delay)
```

Comment:
In this case, would it not make more sense to put the 0.05 as function-argument default? So:
``` suggestion 
    def wait_for(self, query_delay=0.05):
        super().wait_for(query_delay)
```

Reply:
The `values` method calls wait_for with None.
Therefore, the default argument 0.5 would not be used.

Reply:
Ahh check, good point.